storybook/native/stories.js

```diff
@@ -1,11 +1,120 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react-native';
+import styled from 'styled-components/native';
 import Button from '../../src/components/Button/native';
 import Text from '../../src/components/Text/native';
+import theme from '../../src/theme';
+
+const ButtonContainer = styled.View`
+  marginVertical: 8;
+  marginHorizontal: 8;
+`;
+
+const View = styled.View``;
 
 storiesOf('Button', module)
-  .add('with text', () => (
+  .add('default', () => (
     <Button>
       <Text>Default Button</Text>
     </Button>
+  ))
+  .add('kinds', () => (
+    <View>
+      <ButtonContainer>
```

Comment:
Try using the Spacer component here instead of a custom ButtonContainer component here